x/wasm-storage/types/codec.go

```diff
@@ -21,3 +25,32 @@ var (
 	Amino     = codec.NewLegacyAmino()
 	ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
 )
+
+const (
+	b             byte   = 0
+	u             uint64 = 0
+	ByteLenFilter        = int(unsafe.Sizeof(b))
+	ByteLenUint64        = int(unsafe.Sizeof(u))
```

Comment:
Not sure about purpose of these when `ByteLenFilter` and `ByteLenUint64` are fixed values.

Reply:
Initially put 
```
ByteLenFilter        = 1
ByteLenUint64        = 8
```

But it looked ugly. Current approach express the intention more clearly even to a non go reaader.

Reply:
No big deal, but still unsure what this accomplishes.